### DISCUSSION TITLE
How do i set a cronjob to autorun topgrade on boot? (not as easy as you think)

### DISCUSSION THREAD

**Discussion by JamesClarke7283** [topgrade-rs/topgrade#736] [Q&A]
I want to make topgrade run every time i boot my computer, but it finishes with errors(errors i dont know how to capture).

Could you please share with me a way i can run topgrade on a cronjob that works?

**Comment by SteveLauC** [comment]
> but it finishes with errors(errors i dont know how to capture).

I am not familiar with `cron`, what's cron's reaction to Topgrade's error?

**Reply by JamesClarke7283** [reply]
it comes from the cronie package in archlinux, you know `crontab -e` stuff.
I can't seem to capture the error, as ive said. if you attempt to make a cron, with running topgrade, it does not work, even when sudo does not require a password.

**Comment by lucaspar** [comment]
Have you tried redirecting both stdout and stderr to a user-writable log file? Like:

```
topgrade --yes --skip-notify --verbose |& tee -a ~/.cache/topgrade.log
```

also, you might want to `--disable` steps that ask the user's password.

**Comment by GideonBear** [comment]
Replaced by #739
